nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3DirectBatchHandler.java

```diff
@@ -117,10 +133,16 @@ protected void initialize(Map<String, String> properties) {
         region = properties.get(BUCKET_REGION_PROPERTY);
         bucket = properties.get(BUCKET_NAME_PROPERTY);
         bucketPrefix = defaultIfEmpty(properties.get(BUCKET_PREFIX_PROPERTY), "");
-        roleArn = properties.get(ROLE_ARN_PROPERTY);
         accelerateModeEnabled = Boolean.parseBoolean(properties.get(ACCELERATE_MODE_ENABLED_PROPERTY));
         String awsSecretKeyId = properties.get(AWS_ID_PROPERTY);
         String awsSecretAccessKey = properties.get(AWS_SECRET_PROPERTY);
+        expiration = Integer.valueOf(defaultIfEmpty(properties.get(INFO_EXPIRATION), "0"));
```

Comment:
What is the purpose of expiration? It does not seem to be used in the class.

Reply:
`INFO_EXPIRATION` is already used for credentials expiration. Please don't reuse the same constant and property for a different goal. Be explicit, like `INFO_FEDERATION_TOKEN_EXPIRATION = "federationTokenExpiration"`